src/foremast/utils/dns.py

```diff
@@ -148,6 +149,13 @@ def update_failover_dns_record(env, zone_id, **kwargs):
     zone_name = hosted_zone_info['HostedZone']['Name'].rstrip('.')
     dns_name = kwargs.get('dns_name')
 
+    #Check that the primary record exists
+    failover_state = kwargs.get('failover_state')
+    if failover_state.lower() != 'primary':
+        primary_exists = primary_record_exists(env, zone_id, dns_name)
+        if not primary_exists:
+            raise PrimaryDNSRecordNotFound("Primary Failover DNS record not found: {}".format(dns_name))
```

Comment:
No, the issue with that is you are not guaranteed that the primary ELB even exists when creating the secondary record. By just failing instead of creating the primary, I think it avoids some weird corner case issues that would get hacky to work around. I'd also rather keep the runs 'pure', meaning they only impact the region you are actively deploying too. 

I am open to debate on this, those are just my justifications for doing it this way.